packages/edit-site/src/components/global-styles/dimensions-panel.js

```diff
@@ -33,6 +33,7 @@ export default function DimensionsPanel() {
 	const [ inheritedStyle, setStyle ] = useGlobalStyle( '', undefined, 'all', {
 		shouldDecodeEncode: false,
 	} );
+	const [ userSettings ] = useGlobalSetting( '', undefined, 'user' );
 	const [ rawSettings, setSettings ] = useGlobalSetting( '' );
```

Comment:
A non-blocking question: why is this grabbing the root settings via `''` instead of getting the layout settings via `useGlobalSetting( 'layout' )`? Is that so that in follow-ups we can expand the range of settings that folks can manage from the global styles UI, rather than having to have separate `useGlobalSetting` calls for each part of the settings object?

Reply:
No, that's because the underlying component have been refactored to have always an API that receives the whole "settings" object and the whole "style" object.